bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -73,7 +73,8 @@ export default withRouter(injectIntl(withModalMounter(createContainer((
     });
 
     return {
-      sidebar: getCaptionsStatus() ? <ClosedCaptionsContainer /> : null,
+      closedCaption: getCaptionsStatus() ? <ClosedCaptionsContainer /> : null,
+      sidebar: null,
```

Comment:
Just remove sidebar from this list.
We will know that it's still there and pass an element when we need to use it.